package/MDAnalysis/coordinates/PDB.py

```diff
@@ -384,6 +384,8 @@ def _read_frame(self, frame):
             self.convert_pos_from_native(self.ts._unitcell[:3])
         self.ts.frame = frame
         self.ts.data['occupancy'] = occupancy
+        for aux in self.aux_list:
```

Comment:
Can you raise an Issue about how PDB + XTC are different from others so I can look into it.  This is the kind of code I want to avoid as it leads to things diverging once someone tries to update how auxs work (and doesn't realise PDB had its own treatment).